full-service/src/db/transaction_log.rs

```diff
@@ -1069,7 +1069,7 @@ mod tests {
             &wallet_db.get_conn().unwrap(),
         )
         .unwrap();
-        assert_eq!(input_details0.value as u64, 7 * MOB);
+        assert_eq!(input_details0.value, associated_txos.inputs[0].value);
```

Comment:
What caused this change?

Reply:
This should no longer be relevant, that particular code in the test has changed to better reflect the desired outcome